dojo/tools/asff/parser.py

```diff
@@ -41,9 +41,29 @@ def get_findings(self, file, test):
             else:
                 active = False
 
+            # Adding the Resources:0/Id value to the description.
+            #
+            # This is needed because every Finding in AWS from Security Hub has an
+            # associated ResourceId that contains the full AWS ARN and without it,
+            # it is much more difficult to track down the specific resource.
+            #
+            # This is different from the Finding Id - as that is from the Security Hub
+            # control and has no information about the offending resource.
+            #
+            # Retrieve the AWS ARN / Resource Id
+            resource_arn = item.get("ProductFields").get("Resources:0/Id")
```

Comment:
`ProductFields` is an [optional ASFF field](https://docs.aws.amazon.com/securityhub/latest/userguide/asff-top-level-attributes.html#asff-productfields). `Resources`, on the other hand, is a [required field](https://docs.aws.amazon.com/securityhub/latest/userguide/asff-required-attributes.html#Resources) and might make more sense to reference here. You can also get more than one ARN more easily, if applicable to a given finding.

Reply:
Good catch - this is the exact same value, done in a much better way.  Thank you!